test/unit/spec/media/track/remotemediatrack.js

```diff
@@ -59,6 +60,42 @@ const { FakeMediaStreamTrack } = require('../../../../lib/fakemediastream');
       });
     });
 
+    const { trackPriority } = require('../../../../../lib/util/constants');
+
+    describe('.setPriority', () => {
+      let options;
+      let track;
+      let onPriorityChange;
+      beforeEach(() => {
+        onPriorityChange = sinon.spy();
+        track = makeTrack('foo', 'bar', kind, true, options, RemoteTrack, onPriorityChange);
+      });
+
+      [null, ...Object.values(trackPriority)].forEach((priorityValue) => {
```

Comment:
@makarandp0 Test for verifying return value?

Reply:
this test does assert at the end that value equals.